src/app/[locale]/(auth)/login/page.tsx

```diff
@@ -10,9 +17,32 @@ export default function LoginPage() {
   };
 
   return (
-    <main className="h-full w-full bg-gradient">
-      로그인페이지
-      <button onClick={handleGoBack}>뒤로가기</button>
+    <main className="h-full w-full dark-gradient">
+      <div className="flex items-center justify-end p-3 pb-0">
+        <VscChromeClose className="mt-[10px] h-6 w-6 scale-transition-105" onClick={handleGoBack} />
```

Comment:
```suggestion
        <VscChromeClose className="mt-[10px] h-6 w-6 hover:scale-transition-105" onClick={handleGoBack} />
```
hover 시에 커지는 효과 넣어주셔야 합니다.

Reply:
✅ 8916f28e0a62589b09af82bbc581c80930a9f2c0